examples/cmake_android/BUILD

```diff
@@ -1,3 +1,5 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+load("@rules_android//android:rules.bzl", "android_binary", "android_library")
```

Comment:
This seems to be [failing on BuildKite](https://buildkite.com/bazel/rules-foreign-cc/builds/1282#0428871d-995f-45d2-8b16-662c8c404077):

```
(14:10:54) ERROR: error loading package 'cmake_android': Unable to find package for @rules_android//android:rules.bzl: The repository '@rules_android' could not be resolved.
```

Reply:
fixed